### ISSUE TITLE
About Config of pretrained model

### ISSUE THREAD

**Issue by cyxomo** [lmnt-com/wavegrad#3]
this issue presented because there are too much noise in the synthesis audio. I find the pretrained model which applied on 24k audio, however, the default trained config is for 22050 sample rate audio. Are there some difference on config, e.g is noise_schedule np.linspace(1e-4, 0.005, 1000) or Linear(1e−4, 0.005, 1000)

**Comment by cyxomo** [comment]
By the way, there indeed is the gap between the Audio samples in this repository and the samples in https://wavegrad.github.io/, would some parameter be carefully adapted?

**Comment by sharvil** [comment]
The only difference between the default configuration and the pre-trained model's configuration is the `sample_rate`.

It's unclear why there's a gap between the audio samples presented here and the official audio samples at https://wavegrad.github.io. Unfortunately, the paper doesn't provide any results on public datasets or details on training procedure so there's no way to fully reproduce their results.

One difference is that they train with a much larger batch size (256) using TPUv2 pods. Besides that, my suspicion is that their proprietary dataset has much less variance in recording conditions with little noise, so the synthesized audio is also cleaner.

**Comment by CookiePPP** [comment]
> One difference is that they train with a much larger batch size (256) using TPUv2 pods

How do you identify the learning rate?
It needs to be adjusted for the batch size but I don't see a learning rate in the paper so how did you find `2e-4`?

**Comment by sharvil** [comment]
I haven't done any hyperparameter tuning. The learning rate came from the [diffusion repository](https://github.com/hojonathanho/diffusion/blob/master/scripts/run_cifar.py#L134). I also don't know how much hyperparameter tuning they've done.

**Comment by sharvil** [comment]
Closing out due to lack of activity. Feel free to open another issue if you have additional questions.
